ui/app/components/secure-variable-form.hbs

```diff
@@ -16,6 +16,13 @@
         {{autofocus}}
         data-test-path-input
       />
+      {{#if @model.isNew}}
```

Comment:
`thought` `suggestion` `non-blocking`:  This conditional makes me want to rethink the component API. The component is used for new and existing variables and I think we can clean this up to reflect those differences.